component/datamodel/util/write.js

```diff
@@ -0,0 +1,33 @@
+// Copyright IBM Corp. 2015,2016. All Rights Reserved.
+// Node module: loopback-workspace
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+'use strict';
+const fs = require('fs-extra');
+const path = require('path');
+
+class WriteOperations {
```

Comment:
I'm curious as to why you used a class here since you don't `new WriteOperations` in the next file? Why not functional style and just export each operation directly from your utility lib?

```
exports.writeDataSourceConfig = writeDataSourceConfig;
exports.writeModel = writeModel;

function writeModel() {
}
function writeDataSourceConfig() {
}